### DISCUSSION TITLE
What is the approved method of completely uninstalling OpenModelica?

### DISCUSSION THREAD

**Discussion by vtnate** [OpenModelica/OpenModelica#11287] [Q&A] [answered]
For instance, if I want to downgrade to an earlier version for testing/comparison, the [Linux download page](https://openmodelica.org/download/download-linux/) says `make sure all existing OpenModelica packages have been uninstalled (so you do not end up with mismatching versions of dependencies)`.

My current technique is:
- sudo apt remove omc-common
- sudo apt remove omc
- sudo apt remove openmodelica

Is that sufficient? Is that the "right" way to do it? What is the process for uninstalling/downgrading on Windows? Can this be posted on the website somewhere so I don't have to pester you? Perhaps linked from the download page where it says to do that?

Thank you!

**Comment by casella** [comment]
@sjoelund can you comment on that?

**Comment by sjoelund** [comment, marked as answer]
For Linux, that's sufficient. For Windows, you can install multiple versions in parallel for the latest releases.
